dataactvalidator/scripts/load_cfda.py

```diff
@@ -0,0 +1,16 @@
+from ftplib import FTP
+
+ftp = FTP('ftp.cfda.gov')      # connect to host, default port
+ftp.login()                    # user anonymous, passwd anonymous@
+ftp.cwd('usaspending')
+
+data = []
+ftp.dir('-t', data.append)
+file_listing = data[0]
+file_parts = file_listing.split(' ')
+file_name = file_parts[-1]
+print('Loading ' + file_name)
+
+ftp.retrbinary('RETR ' + file_name, open('dataactvalidator/config/cfda_program.csv', 'wb').write)
```

Comment:
The file is actually save as  `cfda_program.csv` to /dataactvalidator/config directory. The new file will replace the old `cfda_program.csv`. We don't need to delete the old file since it saves as a replacement of the old file. I did it that way since we have all other data tables load to this directory prior to load to the database.  It's better to be fully consistent? 

Let me know which one you prefer.